lighthouse.go

```diff
@@ -710,14 +710,10 @@ func (lh *LightHouse) unlockedShouldAddV6(vpnAddr netip.Addr, to *V6AddrPort) bo
 }
 
 func (lh *LightHouse) IsLighthouseAddr(vpnAddr netip.Addr) bool {
-	if _, ok := lh.GetLighthouses()[vpnAddr]; ok {
-		return true
-	}
-	return false
+	_, ok := lh.GetLighthouses()[vpnAddr]
+	return ok
 }
 
-// TODO: CERT-V2 IsLighthouseAddr should be sufficient, we just need to update the vpnAddrs for lighthouses after a handshake
```

Comment:
I think it is more burdensome to do that? This feels fine to me, but I'm open to making the change if you feel differently

Reply:
The goal of this TODO was to remove the loop and made a stab at how we might do it. 

#1464 gave us a more reliable `lh.addrMap` for non primary vpn addrs in the static host map and a TODO in that PR was to update the `RemoteList.vpnAddrs` once we received and authenticated the certificate from the remote (to fix the `remote_allow_ranges`). With that, we would have a way to kill this loop.

Reply:
Not specifically, it is a performance regression in some ways but I think it has low impact.